src/types/quizzes.d.ts

```diff
@@ -9,6 +20,13 @@ export interface QuizzesParameters {
   versionId?: string;
 }
 
+export interface QuizzesResultsParameters extends QuizzesParameters {
+  answers: any[];
```

Comment:
answers already part of QuizzesParameters did you intend to have it here also

Reply:
They are optional there but required here. Is this the right way to do it?

Reply:
Looks right to me didn't notice that it's required here